lab-khalid/lib/error-middleware.js

```diff
@@ -0,0 +1,16 @@
+const createError = require('http-errors');
+const debug = require('debug')('employee:error-middleware');
+
+module.exports = function(err, req, res, next){
+  debug('error-middleware');
+  if(err.status){
+    debug('User error');
+    res.status(err.message).send(err.message);
```

Comment:
You are currently passing the err.message where you should actually be passing err.status.

**Be very very careful when copying and pasting old code. This was an error you had in your last code submission.**